packages/admin/cms-admin/src/dependencies/DependencyList.tsx

```diff
@@ -0,0 +1,229 @@
+import { TypedDocumentNode, useApolloClient, useQuery } from "@apollo/client";
+import { Tooltip, useDataGridRemote } from "@comet/admin";
+import { ArrowRight, OpenNewTab, Reload } from "@comet/admin-icons";
+import { IconButton } from "@mui/material";
+import { LabelDisplayedRowsArgs } from "@mui/material/TablePagination/TablePagination";
+import { GridColDef } from "@mui/x-data-grid";
+import * as React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import { useHistory } from "react-router";
+
+import { useContentScope } from "../contentScope/Provider";
+import { GQLDependency } from "../graphql.generated";
+import { useDependenciesConfig } from "./DependenciesConfig";
+import * as sc from "./DependencyList.sc";
+import { DependencyInterface } from "./types";
+
+export type DependencyItem = Pick<GQLDependency, "name" | "secondaryInformation" | "rootColumnName" | "jsonPath"> & {
+    id: string;
+    graphqlObjectType: string;
+};
+
+type Dependency = {
+    targetGraphqlObjectType: string;
+    targetId: string;
+    rootColumnName: string;
+    jsonPath: string;
+    name: string | null;
+    secondaryInformation: string | null;
+};
+
+type Dependent = {
+    rootGraphqlObjectType: string;
+    rootId: string;
+    rootColumnName: string;
+    jsonPath: string;
+    name: string | null;
+    secondaryInformation: string | null;
+};
+
+interface GQLQuery {
```

Comment:
Please don't use the GQL prefix for this types, as there indicating that a type was generated